bin/birdhouse

```diff
@@ -149,6 +150,7 @@ Options:
   -a, --component-application-data    Restore application data for a given component (e.g. magpie). This can be specified multiple times. Use '*' to select all components.
   -u, --component-user-data           Restore user data for a given component (e.g. cowbird). This can be specified multiple times. Use '*' to select all components.
   -l, --component-logs-data           Restore logs data for a given component (e.g. thredds). This can be specified multiple times. Use '*' to select all components.
+  -r, --representative-data           Restore representative data for a given component (e.g. stac). This can be specified multiple times. Use '*' to select all components.
```

Comment:
I never noticed but we have duplicate identical help text?

Reply:
OH sorry, read too fast, it's the exact same text except "Backup" becomes "Restore" so yes they are not identical.